### ISSUE TITLE
Obfuscated download url in computational services

### ISSUE THREAD

**Issue by wvangeit** [ITISFoundation/osparc-simcore#5979]
It would be best to obfuscated part of the url of 
'Download of 'https://s3.amazonaws.com/  ...'
when computational services are downloading their input

**Comment by sanderegg** [comment]
yep. there is also another issue where the computational backend would use the osparc-api. that would probably also help.
